akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -203,11 +204,14 @@ object HttpEntity {
    *
    * If the given `chunkSize` is -1 the default chunk size is used.
    */
-  def apply(contentType: ContentType, file: File, chunkSize: Int = -1): UniversalEntity = {
-    val fileLength = file.length
+  def apply(contentType: ContentType, file: File, chunkSize: Int = -1): UniversalEntity =
```

Comment:
Did we decide about deprecation?
I think we should add deprecation of the `File` methods so that we eventually can remove them and only have `Path`.